lib/crystalball/rails/map_generator/i18n_strategy.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'crystalball/map_generator/base_strategy'
+require 'crystalball/map_generator/helpers/path_filter'
+require 'crystalball/rails/map_generator/i18n_strategy/simple_patch'
+
+module Crystalball
+  module Rails
+    class MapGenerator
+      # Map generator strategy to build map of locale files used by an example.
+      class I18nStrategy
+        include ::Crystalball::MapGenerator::BaseStrategy
+        include ::Crystalball::MapGenerator::Helpers::PathFilter
+
+        class << self
+          # List of locale files affected by current example
+          #
+          # @return [Array<String>]
+          attr_reader :locale_files
+
+          # Reset cached list of locale files
+          def reset_locale_files
+            @locale_files = []
+          end
+        end
+
+        def after_register
+          SimplePatch.apply!
+        end
```

Comment:
its not required to be after register. It can be `after_start`

Reply:
It is required. I18n loads files at the moment we configure it. what will be before we initialize rspec suit